README.md

```diff
@@ -10,6 +10,11 @@ Valid version:
 - 1.0.0  Change: initial version. Limitation: cannot install with pecl on linux; cannot work with 7.2.23+ and 7.3.10+
 - 1.0.1  Change: with pecl install command line support on linux. Limitation: cannot work with 7.2.23+ and 7.3.10+
 - 1.0.2  Change: fix compatibility problem with  7.2.23+ and 7.3.10+
+- 1.0.3  Change: In preious versions, if connection doesnot use SSL, or server doesnot support redirection, or redirected connection fails to connect for some reason, it will fallback to the first proxy connection. Since 1.0.3, the logic changes as follows: 
```

Comment:
Nit: 
```suggestion
- 1.0.3  Change: In previous versions, if connection does not use SSL, or server does not support redirection, or redirected connection fails to connect for some reason, it will fallback to the first proxy connection. Since 1.0.3, the logic changes as follows: 
```

Reply:
Hi Christoph,

Thanks a lot for your detailed check. I've enhanced the document and change it accordingly.